src/lib/data.js

```diff
@@ -55,7 +55,12 @@ export const mappings2018 = {
 .0060,
 .0045,
 .0029,
-.0069,]
+.0069,],
```

Comment:
I feel this would look a bit neater like [this](https://github.com/Larry-Gan/PCSTaxToolV2/blob/main/src/lib/data.js#L58). Guess this applies to everything else in this file as well.